src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -954,6 +954,8 @@ def load_separator_node_properties_description(
         "initial_phase_volume_fractions": get_dict_with_scalar_loader(
             category="volume fraction"
         ),
+        "gas_separation_efficiency": get_scalar_loader(from_unit="%"),
```

Comment:
I believe you have to make something similar in the `case_to_alfacase.py` file.
@williamjamir could you confirm?

Reply:
Also, would be nice to add tests for the new entries.

Reply:
@williamjamir, I do not remember, do we need to add tests? Or the test suit makes it automatically for any new entry?

Reply:
No. `case_to_alfacase` is almost automatic.
I hope in the future to mate a bit less reliant o manual updates =)

Reply:
> I believe you have to make something similar in the case_to_alfacase.py file.

Case to ALFAcase is more "smart", is just a loop over all attributes, and automatically gets the value.
You will need to update that only if a new "type" is added.

> Also, would be nice to add tests for the new entries.
Definitely, please update the `SeparatorNodePropertiesDescription` on `alfasim-sdk/tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py` and add these attributes there as well.

> @williamjamir, I do not remember, do we need to add tests? Or the test suit makes it automatically for any new entry?

There is a basic "round-trip" test that takes a filled case description and converts it to ALFAcase and reimport to check if the values match.
With this test, you can validate that the conversion (ALFAcase->case; case->ALFAcase) is working.

Reply:
Thanks a lot, Guys!

@edersongrein if you don't understand some of the instructions, please let me know, I can help you on that.

Reply:
We didn't add (yet) a smart way to automatically check if the filled case description is outdated (that would be nice).

I think that one possible solution is to add a "template" when a new PR is opened with TODOS to be check and one of the options is "Did you update the filled case description available at X ? ".

Reply:
Sounds good to me. I think it would help.

Reply:
> Thanks a lot, Guys!
> 
> @edersongrein if you don't understand some of the instructions, please let me know, I can help you on that.

Thanks. Probably I will